minecraft/src/main/java/org/quiltmc/loader/impl/game/minecraft/MinecraftGameProvider.java

```diff
@@ -474,6 +474,8 @@ public void launch(ClassLoader loader) {
 			targetClass = "org.quiltmc.loader.impl.game.minecraft.applet.AppletMain";
 		}
 
+		Log.log(LogCategory.GAME_PROVIDER, "Launching using target class '" + targetClass + "'");
```

Comment:
Oh er you'll need to add an import to `org.quiltmc.loader.impl.util.log.LogCategory` too